### DISCUSSION TITLE
how to disconnect websocket when switch url to another node

### DISCUSSION THREAD

**Discussion by DogLi** [Kong/kong#13596] [Help]
I bind a domain name to a node and there is a websocket connected to the node. When I switch the domain name to another node, the websocket is not disconnected. How to disconnect it and reconnect to domain url(connect the new node)?

**Comment by chobits** [comment]
The question description is too general, and we are unable to ascertain the specific issue that has occurred. Could you provide detailed steps to reproduce your problem, including specific commands?
